### DISCUSSION TITLE
Command-line startup arguments

### DISCUSSION THREAD

**Discussion by alikve** [Jelmerro/Vieb#548] [General]
`vieb --help` reads:

`Args not starting with '-' will be opened as in a new tab if it's a valid url`

It seems that the "valid url" here must start with `http(s)`.
If I run `vieb http://example.org`, Vieb opens a new tab as expected, but if I run `vieb example.org`, Vieb writes
`Sending urls to existing instance /home/slalik/.config/Vieb/`
but nothing happens.

In the main documentation, in the description of Explore mode it says that valid url is indicated by blue color and by testing I see that the blue color doesn't require http(s).

I'd suggest, if possible, that args (not starting with '-') be opened in a new tab as if they were entered in the Explore mode, including search if they are not url.

**Comment by Jelmerro** [comment]
This is a bug, but a breaking change to fix, so I'll make sure this will work in the upcoming V12 release. Thanks for letting me know this is not working well.

**Comment by Jelmerro** [comment]
Implemented in 12.0.0 :tada:
